ci-operator/config/openshift/machine-api-provider-azure/openshift-machine-api-provider-azure-main.yaml

```diff
@@ -0,0 +1,88 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.9:
```

Comment:
Should this, and others, be updated to 4.10?

Reply:
I think it will be updated automatically in some time